### DISCUSSION TITLE
Testing Idena node: release candidate 0.27.2-rc1

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#812] [General]
We invite you to test the release candidate version of the Idena node: 0.27-2-rc1.
Please [download ](https://github.com/idena-network/idena-go/releases/tag/v0.27.2-rc1) it manually if you'd like to join the testing.

Changes:
- Performance improvements

We plan to release these changes in two days unless negative feedback is received

**Comment by rioda-org** [comment]
I have tested this node version on two locations. I had them running without identities, so new addresses. Both on Windows 10, ssd.

Location 1 - had two more nodes running version 0.27.1 on the same internet connection. I've experienced occasional network connectivity issues which you will see in log file. I'm not sure if node caused LAN connection to reset only for my PC or did something to the router, or router did something (Cisco ASA 5505). But as soon as I've stopped node, disconnections stopped.
[location1.zip](https://github.com/idena-network/idena-go/files/7278314/location1.zip)

Location 2 - this was the only running node on this connection, simple home cable router, everything was smooth, no disconnections here.
[location2.zip](https://github.com/idena-network/idena-go/files/7278315/location2.zip)

Memory managment is superb. Node keeps to ~320 MB while doing fast sync. When it comes to last few hundred of blocks, it goes up to 650 MB, but as soon it's finished and synced, it starts to clear memory and brings itself back to 300 MB range.

**Reply by midenaio** [reply]
Thanks a lot for you feedback
